pkg/client/kubernetes/client.go

```diff
@@ -181,23 +181,6 @@ func NewForConfig(config *rest.Config, options client.Options) (Interface, error
 		machine:         machine,
 		apiregistration: apiRegistration,
 		apiextension:    apiExtension,
-
-		resourceAPIGroups: map[string][]string{
```

Comment:
Can you also remove this field from the struct in `pkg/client/kubernetes/types.go`?